package.json

```diff
@@ -44,7 +44,9 @@
     "node": ">= 0.10.0"
   },
   "scripts": {
-    "test": "grunt coverage"
+    "test": "grunt coverage",
```

Comment:
From my point of view - there is always a little space for small improvements even if they are not related very much to the issue scope. An issue requesting improvements like this one will never have high priority because there are always more priority bugs or feature requests and thus they would never be done.

Therefore I think that these little changes can be done as a part of a bigger PR so they won't waste programmers time to create an issue, PR and PR reviews for a 2 line change and with this, the project will keep constantly improving.